.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,11 +1,11 @@
-name: Feature Request
-description: Suggest an idea for this project
+name: Feature/Balance/Change Request
```

Comment:
What about just `Change Request`? I've also seen `Enhancement Request` in professional development settings.

Reply:
Changes, and more-so Enhancements, still sound more specific than including the three examples in this PR. Changes aren't always the same thing as Feature requests, Balance changes, etc..